x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/playwright.config.ts

```diff
@@ -10,4 +10,5 @@ import { createPlaywrightConfig } from '@kbn/scout';
 // eslint-disable-next-line import/no-default-export
 export default createPlaywrightConfig({
   testDir: './tests',
+  workers: 2,
```

Comment:
Should this config have a single worker?

Reply:
yes, thanks for checking.